e2e/data-factory/tests/post-factory.test.ts

```diff
@@ -0,0 +1,95 @@
+import {test, expect} from '@playwright/test';
+import knex, {Knex} from 'knex';
+import {PostFactory} from '../factories/posts/post-factory';
+import {KnexPersistenceAdapter} from '../persistence/adapters/knex';
+
+// Database configuration - uses Ghost's development database
+const dbConfig = {
+    client: process.env.database__client || 'mysql2',
+    connection: {
+        host: process.env.database__connection__host || '127.0.0.1',
+        user: process.env.database__connection__user || 'root',
+        password: process.env.database__connection__password || 'root',
+        database: process.env.database__connection__database || 'ghost'
+    }
+};
+
+test.describe('PostFactory', () => {
+    let db: Knex;
+    let postFactory: PostFactory;
+    let adapter: KnexPersistenceAdapter;
+
+    test.beforeAll(async () => {
+        // Initialize database connection
+        db = knex(dbConfig);
+        
+        // Verify connection
+        await db.raw('SELECT 1');
+        
+        // Initialize adapter and factory
+        adapter = new KnexPersistenceAdapter(db);
+        postFactory = new PostFactory();
+        postFactory.setAdapter(adapter);
+    });
+
+    test.afterAll(async () => {
+        if (db) {
+            await db.destroy();
+        }
+    });
+
+    test.afterEach(async () => {
+        // Clean up test data
+        if (db) {
+            await db('posts').where('title', 'like', '%Test%').del();
+        }
+    });
+
+    test('should build a post in memory without persisting', async () => {
+        const post = postFactory.build({
+            title: 'Test Post',
+            status: 'draft'
+        });
+        
+        expect(post.id).toBeTruthy();
+        expect(post.title).toBe('Test Post');
+        expect(post.status).toBe('draft');
+        expect(post.slug).toBeTruthy();
+        expect(post.html).toBeTruthy();
```

Comment:
I think we don't need to test these, we will exercise the tests anyways in e2e tests. We could leave the happy path one, but not really needed - we will duplicate most of what we will do in e2e test